lms/views/status.py

```diff
@@ -11,7 +12,11 @@
 def status(request):
     try:
         request.db.execute(text("SELECT 1"))
-        return {"status": "okay"}
     except Exception as err:
         LOG.exception("Executing a simple database query failed:")
         raise HTTPInternalServerError("Database connection failed") from err
+
+    if "sentry" in request.params:
+        capture_message("Test message from LMS's status view")
```

Comment:
This could be even more end-to-end if instead of explicitly sending a message to sentry it just raised an exception. That should make its way to sentry and return an error in the view.